src/configure/clients/github/TemplateServiceClient.ts

```diff
@@ -1,19 +1,505 @@
+import { ExtendedPipelineTemplate } from "../../model/Contracts";
 import { RepositoryAnalysisParameters } from "../../model/models";
 import { PipelineTemplateMetadata } from "../../model/templateModels";
 import { RestClient } from "../restClient";
 
 export class TemplateServiceClient {
     private restClient: RestClient;
+    private  readonly templateServiceUri: string = "https://tswithouthmac.azurewebsites.net/Templates";
```

Comment:
This should be the PE service wala url. Lets us try not use the temporary server anymore.

Reply:
done